src/jdk.incubator.vector/linux/native/libjsvml/globals_vectorApiSupport_linux.S.inc

```diff
@@ -4,7 +4,9 @@
  *
```

Comment:
Please update the copyright year to 2022.

Reply:
:+1: